psd-web/spec/features/report_product_spec.rb

```diff
@@ -317,14 +448,19 @@ def expect_case_activity_page_to_show_test_result(test)
     expect(item).to have_text(test[:details])
   end
 
+  def fill_in_coronavirus_page(answer)
```

Comment:
Suggest renaming of the helper method:

```suggestion
  def answer_is_case_coronavirus_related_with(answer)
```

Reply:
I kept the naming consistent with all of the other `fill_in_page` helpers so I'd prefer to keep it the same for consistency. We can look at overall naming consistency across all tests separately.